src/renderer/style.scss

```diff
@@ -252,11 +252,50 @@ $sceneDetailOptionsWidth: 220px;
   padding: $smallPadding;
 }
 
-.SimpleOptionPicker {
-  padding-bottom: 9px;
-
-  label {
-    margin-right: 0.5rem;
+.SceneOptionsForm {
+  div {
+    padding-bottom: 9px;
+    label {
+      margin-right: 0.5rem;
+    }
+  }
+  .SimpleTextInput {
+    input[type=text] {
+      font-size: 16px;
+      font-weight: 400;
+      display: block;
+      width: 100%;
+      margin-bottom: 0;
+    }
+  }
+  .SimpleSliderInput {
+    input[type=range] {
+      -webkit-appearance: none;
+      width: 100%;
+      height: 10px;
+      border-radius: 5px;
+      outline: none;
+      -webkit-transition: opacity .2s ease-in-out;
```

Comment:
FYI, you generally don't need `-webkit-` on anything. This stuff is well supported now. Don't worry about removing it if you don't feel like it.